doc/dotnet-support.md

```diff
@@ -0,0 +1,39 @@
+# Mystikos Support for .Net
+
+### Supported .Net and ASP.Net version
+We recommend using .net/ASP.Net 3.1 and .net/ASP.Net 5.0.
+
+### Supported Distros:
+- Alpine
+- Ubuntu
+
+We recommend using official images from [mcr.microsoft.com/dotnet](https://hub.docker.com/_/microsoft-dotnet/).
+
+### Recommended configurations
+
+#### dotnet configurations
+- Run with dotnet diagnostics disabled. This can be controlled via the `COMPlus_EnableDiagnostics` environment variable set to 0.
+- If running with low memory settings, you can control dotnet's managed heap size by [`COMPlus_GCHeapHardLimit` environment variable](https://docs.microsoft.com/en-us/dotnet/core/run-time-config/garbage-collector#heap-limit).
```

Comment:
As discussed in the meeting, we should also experiment with  building the dotnet app with or without ``self-contained` flag and see the impacts on the memory requirement.

Reply:
Ignore the previous comment. My experiment shows `self-contained` is not relevant to memory requirement.

Reply:
Resolving based on your Teams message that `self-contained` doesn't impact memory requirement.